code/framework/src/scripting/engines/node/builtins/color_rgb.h

```diff
@@ -0,0 +1,109 @@
+/*
+ * MafiaHub OSS license
+ * Copyright (c) 2021-2023, MafiaHub. All rights reserved.
+ *
+ * This file comes from MafiaHub, hosted at https://github.com/MafiaHub/Framework.
+ * See LICENSE file in the source repository for information regarding licensing.
+ */
+
+#pragma once
+
+#include "../../../keys.h"
+
```

Comment:
Do we need two separate classes to represent colors? I'd suggest adapting ColorRGBA to cover all ColorRGB needs.

Reply:
I think so, because on the scripting side, we either want to manipulate an RGB or an RGBA.

Reply:
People can use RGBA to represent RGB colors, too, we will just ensure alpha is always 255 in such cases. It just feels like there's too much code duplication to represent the same thing in JS context.